update.sh

```diff
@@ -3,13 +3,15 @@ set -e
 
 declare -A gpgKeys
 gpgKeys=(
+	[7.0]='1A4E8B7277C42E53DBA9C7B9BCAA30EA9C0D5763'
```

Comment:
@yosifkit I think it's better to leave the update.sh generate the apache/fpm files from the 7.0/Dockerfile If I remove that file and use the pristine update.sh we will have to manually mantain the 7.0 Dockerfiles